packages/rocketchat-livechat/client/route.js

```diff
@@ -19,6 +19,22 @@ AccountBox.addRoute({
 	pageTemplate: 'livechatCurrentChats'
 }, livechatManagerRoutes);
 
+AccountBox.addRoute({
+	name: 'livechat-analytics',
+	path: '/livechat-analytics',
```

Comment:
Let's remove the prefix `livechat` in `path` prop, like this -> `path: '/analytics'`.